data/society/public-safety/index.html

```diff
@@ -109,6 +118,20 @@ <h3 id="LawEnforcement">Law Enforcement</h3>
     info=page.LawEnforcement
     %}
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="LawEnforcementBoundaries">Law Enforcement Boundaries</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/Lawboundaries.png %}" alt="Law Boundary icon" loading="lazy" />
+    <p>Society.LawEnforcementBoundaries contains jurisdictional boundaries for law enforcement agencies in Utah. Law enforcement boundaries indicate which law enforcement agency has the primary responsibility in a specific area. These boundaries are also used in the Next-Generation 911 system for recommending which agency should respond to a specific location.</p>
```

Comment:
Nitpick: These are the internal table names. We've been moving away from those by using the normal name.

Reply:
@steveoh Okay, I guess I was just trying to align it with the way the rest of the page's tables were named.  Should I change all of them to just 'Fire Stations', 'PSAP Boundaries', 'Law Enforcement Boundaries', etc.?

Reply:
Yah that would be an improvement